src/main/java/com/blamejared/crafttweaker/impl_native/util/ExpandResourceLocation.java

```diff
@@ -17,6 +17,7 @@
     public static final String ZC_CLASS_NAME = "crafttweaker.api.util.MCResourceLocation";
     
     @ZenCodeType.Method
+    @ZenCodeType.Operator(ZenCodeType.OperatorType.COMPARE)
```

Comment:
Could you check the NativeConverter to see if we already handle Operators in Expansion methods?
I don't remember if I added it.

Reply:
it works.